assessment.md

```diff
@@ -361,17 +361,43 @@ TODO: add description for each assessment level for this skill.
 ### Interactions
 
 * Interaction with the customers
-
-TODO: add description for each assessment level for this skill.
+- SERVICE
+- Social
+- Eager
+- Replace
+- Verbal Communication
+- Informative
+- Countability
+- Easy
+
+The objective for the interaction with a customer is supposed to be smooth and easy. Interacting with customers is one of the most important parts of software engineering. The acronym of "SERVICE" is a great way for us to see what attributes and tendencies lead to good service.
+
+* :flushed: = Refuses to interact with and/or work with customers. Ignores specifics and details regarding the tool from the customer.
+* :worried: = Interacts with and occasionally communicates with the customer, but does not actively participate in completing tasks addressed by the customer.
+* :neutral_face: = Interacts with and works with the customers actively while completing most of the tasks assigned to them by the customer
+* :blush: = Interacts with, works with, and actively collaborates with the customer. Completes all tasks assigned to them by customer.
+* :laughing: = Interacts with, works with, and actively collaborates with the customer. The group of individual are always open to critique and additions from the customer regarding functionality and tools. Completes all tasks assigned to them by the team leader(s) and offers additional help to expedite the completion of other related tasks to complete the project.
```

Comment:
One issue that was not picked out earlier: the indentation here is wrong. Do a tab (I think) on each level, and they should be good. The same applies to the rest of the sections and the acronym. Thanks very much!